lib/krikri/logger.rb

```diff
@@ -0,0 +1,38 @@
+module Krikri
+  ##
+  # An application-wide tagged logger
+  # 
+  # @example with default logger
+  #   Krikri::Logger.log :info, 'message'
+  #
+  # @example with custom logger
+  #   logger = Logger.new(my_logger)
+  #   logger.log :info, 'message'
+  class Logger
+    ##
+    # @param [ActiveSupport::TaggedLogging] logger
+    def initialize(logger = ActiveSupport::TaggedLogging.new(Rails.logger))
```

Comment:
Right. The yard doc for the parameter also asserts this. You could do, e.g. `Logger.new(ActiveSupport::TaggedLogging.new(some_logger)`, or implement a null logger that just eats the messages, or whatever. :)